src/summary/cards/horizontal-label-value.component.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import styles from "./horizontal-label-value.css";
+
+export default function HorizontalLabelValue(props) {
+  return (
+    <div className={styles.root} data-testid="horizontalLabelValue">
```

Comment:
The test id doesn't appear to be used by anything. Also test ids are discouraged by react testing library except for exceptional cases.

```suggestion
    <div className={styles.root}>
```